src/selectors/__tests__/exportContext.ts

```diff
@@ -0,0 +1,78 @@
+import { initialState, reducerFlow } from '../../util'
+import { newThought, setCursor, importText } from '../../reducers'
+import { HOME_PATH } from '../../constants'
+import exportContext from '../exportContext'
+
+describe('exportContext', () => {
```

Comment:
You can omit the outer `describe` since Jest groups the tests by filename.

Reply:
Removed